src/client/web/webKwil.ts

```diff
@@ -105,29 +106,31 @@ export class WebKwil extends Kwil<EnvironmentType.BROWSER> {
   ): Promise<Message> {
     const name = !actionBody.name && actionBody.action ? actionBody.action : actionBody.name;
 
-    // pre Challenge message
-    let msgBuilder = ActionBuilderImpl.of<EnvironmentType.BROWSER>(this)
-      .chainId(this.chainId)
-      .dbid(actionBody.dbid)
-      .name(name)
-      .description(actionBody.description || '');
-
+    let inputs;
     if (actionBody.inputs) {
-      const inputs =
+      inputs =
         actionBody.inputs[0] instanceof ActionInput
           ? (actionBody.inputs as ActionInput[])
           : new ActionInput().putFromObjects(actionBody.inputs as Entries[]);
-      msgBuilder = msgBuilder.concat(inputs);
     }
```

Comment:
@KwilLuke need some guidance with this type assertion. I've tried a lot of troubleshooting but can't seem to make it work by removing as Entries[]. Which is odd, because if you look in kwil.execute(), you'll see that doesn't need type assertions for the inputs logic...